config/config_pcdseg_all_roofs.yaml

```diff
@@ -6,6 +6,7 @@ retrieve_point_clouds.py:
   input_tiles: LiDAR/Emprises tuiles Lidar 2019.shp
   roofs: SHP_CAD_BATIMENT_HORSOL_TOIT/CAD_BATIMENT_HORSOL_TOIT_subset.shp
   overwrite: False
+  tile_id: fme_basena
```

Comment:
I find this parameter a little difficult to understand without context. Perhaps you could change the name of the attribute or add a small explanation as a comment. 
Can this parameter be modified? If not, maybe you could leave it hard-coded or define it as the column number in the script.